src/Enzo/fortran_types.h

```diff
@@ -2,8 +2,8 @@
 #ifdef FORTRAN
 #  ifdef CONFIG_PRECISION_SINGLE
 #     define CONFIG_BFLOAT_4
-#     define tiny 1.d-20
-#     define huge 1.d+20
+#     define tiny 1.e-20
+#     define huge 1.e+20
```

Comment:
Why the change only for single prec but not double or quad (asking as a non-Fortran person)?